app/src/main/java/com/example/weatherandroidapp/views/WeatherGridFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.weatherandroidapp.views
+
+import androidx.lifecycle.ViewModelProvider
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.weatherandroidapp.R
+import com.example.weatherandroidapp.viewmodels.WeatherGridViewModel
+
+class WeatherGridFragment : Fragment() {
+
+    private lateinit var viewModel: WeatherGridViewModel
```

Comment:
Why here you do not use koin to inject the view model as you do it in the other fragment?

Reply:
Done